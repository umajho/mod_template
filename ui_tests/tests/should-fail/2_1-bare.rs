fn main() {}

#[mod_template::define(define_foo)]
fn an_fn() {}
