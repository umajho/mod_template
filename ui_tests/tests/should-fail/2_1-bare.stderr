error: this attribute can only be applied to a `mod`
 --> tests/should-fail/2_1-bare.rs:3:1
  |
3 | #[mod_template::define(define_foo)]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `mod_template::define` (in Nightly builds, run with -Z macro-backtrace for more info)
